src/Controller/ErrorController.php

```diff
@@ -55,4 +55,14 @@ public function beforeRender(Event $event)
 
         $this->viewBuilder()->templatePath('Error');
     }
+
+    /**
+     * afterFilter callback.
+     *
+     * @param \Cake\Event\Event $event Event.
+     * @return void
```

Comment:
Yeah, I agree.
The problem with invalid voids is that the usage/check on those methods become yellow in IDEs like PHPStorm (for a good reason). As you should never have to save the return value of such a method into a variable or use it in the code.
We are doing that for these callbacks though inside the core code.
That clearly points out that those cannot be void by design.